cypress/e2e/Shopping_main_site.cy.js

```diff
@@ -0,0 +1,95 @@
+describe('shopping_main_site', () => {
+    
+    
+    it ('go_to_details_by_name',()=> {
+
+
+
+        cy.visit('/').url().should('eq', 'https://www.saucedemo.com/')
+        cy.login('standard_user', 'secret_sauce')
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+        const go_to_3=cy.get('[id="item_3_title_link"]')
```

Comment:
You shouldn't use const like this to assign elements:
https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Return-Values

It might work for you right now but in more complex cases it won't,

Reply:
done